src/models/ControlMap.ts

```diff
@@ -13,6 +13,13 @@ export class ControlMap {
     private readonly grid: Float32Array;
     private texture: WebGLTexture;
     private gl: WebGLRenderingContext;
+    // catalog spatial matching for webgl2
+    minPoint2: Point2D;
```

Comment:
@panchyo0 doesn't it make more sense to just use a separate control map (and perhaps a separate control map `Map`) for catalogs? The requirements are different: 
- The original control maps are required when viewing contours of one image on another
- The new control maps are required when viewing catalogs of one image on another

We could make a `CatalogControlMap` that extends `ControlMap`, that might make things neater